.travis.yml

```diff
@@ -17,9 +17,9 @@ env:
     - INSTALL_DIR="${TRAVIS_BUILD_DIR}/venv"
     - LD_LIBRARY_PATH="${INSTALL_DIR}/lib:${INSTALL_DIR}/lib64:${LD_LIBRARY_PATH}"
   matrix:
-    - TEST_SUITE="-LE SLOW"  # Run all tests not labeled as slow
-    - TEST_SUITE="-L SLOW_1" # Run all tests labeled as SLOW in group 1
-    - TEST_SUITE="-L SLOW_2" # Run all tests labeled as SLOW in group 2
+    - TEST_SUITE=""  # Run all tests not labeled as slow
+    #- TEST_SUITE="-L SLOW_1" # Run all tests labeled as SLOW in group 1
```

Comment:
yes, as we are more people working on libres and ert, we need to reduce the matrix size on travis, otherwise we will have prs waiting for resources "forever". Will remove the commented out lines